apps/website/src/components/utils/MoveOnVisible.astro

```diff
@@ -0,0 +1,99 @@
+---
+---
```

Comment:
could add some comment what this file is for and how to use it

Reply:
I tried looking for a way to add something similar to JSDocs in Astro, so that hovering over `<MoveOnVisible/>` in a code editor would show the docs. But I couldn't find how to do this.

So I just added a normal JSDocs comment with some info. I can remove some parts of the JSDocs if it's unnecessary info.

Reply:
i meant comments about what data attributes to put where. currently those two bullet points dont really add much

Reply:
Updated [`b4d38c3` (#1032)](https://github.com/iTwin/iTwinUI/pull/1032/commits/b4d38c3d9a8c64cd4f736fd20be3bd336b8e15cf)